cfglib.py

```diff
@@ -170,6 +170,10 @@
                      'migration process', default=False),
     cfg.BoolOpt('hide_ssl_warnings', default=False,
                 help="Don't show ssl warnings"),
+    cfg.StrOpt('src_cinder_conf', default='/etc/cinder/cinder.conf',
```

Comment:
We already have `src_storage_opts` and `dst_storage_opts` for keeping all cinder-related options. Please move these there.